omnibus/package-scripts/agentless-scanner-deb/postrm

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# Perform necessary datadog-agentless-scanner removal steps after package is uninstalled.
+#
+# .deb: STEP 3 of 5
+
+INSTALL_DIR=/opt/datadog/agentless-scanner
+LOG_DIR=/var/log/datadog
+CONFIG_DIR=/etc/datadog-agent
+
+# Remove the symlink to the binary.
+rm -f "/usr/bin/datadog-agentless-scanner"
+
+set -e
+
+case "$1" in
+    purge)
+        echo "Deleting dd-agent user"
+        deluser dd-agent --quiet
+        echo "Deleting dd-agent group"
+        (getent group dd-agent >/dev/null && delgroup dd-agent --quiet) || true
+        echo "Force-deleting $INSTALL_DIR"
```

Comment:
Wouldn't this break if we uninstall & purge the agentless-scanner but not the agent?